cli/src/trusted_base_cli/commands/verify_proof.rs

```diff
@@ -15,76 +15,49 @@
 
 */
 
-use crate::{
-	get_layer_two_nonce,
-	trusted_cli::TrustedCli,
-	trusted_command_utils::{get_identifiers, get_pair_from_str},
-	trusted_operation::perform_trusted_operation,
-	Cli,
-};
-
+use crate::{trusted_cli::TrustedCli, Cli};
 use binary_merkle_tree::verify_proof;
-use codec::{Decode, Encode};
-use ita_stf::{Index, TrustedCall, TrustedGetter, TrustedOperation};
-use itp_stf_primitives::types::KeyPair;
-use log::{debug, info};
-use sp_core::{Pair, H256};
-
-
-use core::hash::Hasher;
-
-use codec;
+use ita_stf::MerkleProofWithCodec;
+use log::info;
+use primitive_types::H256;
+use sp_runtime::traits::Keccak256;
 
 #[derive(Parser)]
 pub struct VerifyMerkleProofCommand {
-	merkle_root: String,
-	merkle_proof: String,
-	orders_encoded_len: usize,
-	leaf_index: usize,
-	leaf: String,
+	merkle_proof_json: String,
```

Comment:
Is that possible to run the `verify_proof` from the CLI?
I see the command accepts a `proof` of type `String`.
But when I try to pass the string proof, it results in an error:

```
./integritee-cli trusted --mrenclave FmLvUktzLZcL8BizX694wecB7nAfb95m6YPPzXa6ypKm --direct verify-proof //Alice "{\"root\":\"0xd889ae8b2b8807ac5845cee7bb8db1996e01554f88df2040353ea0a582df392f\",\"proof\":[\"0x70997831bb3b0a5eb10dd2c55972f35bcbe781d014ad53811d31743450587292\",\"0x91cfc94f5515165bb59ff651be8bbf3058865f393d78bb102ff686b1782ce360\",\"0xccfd66ee8c896087af65b4ae3f577f86ef82092168e61cbf7f08d3951ef1852a\",\"0xff00158da139d439fade205829bfd1153da3b32b45789f567c0898d563d2abce\"],\"number_of_leaves\":10,\"leaf_index\":1,\"leaf\":[1,0,0,0,0,0,0,0,1,100,50,48,50,50,45,48,51,45,48,52,84,48,53,58,48,54,58,48,55,43,48,48,58,48,48,28,97,99,116,111,114,95,49,1,0,0,0,0,154,153,153,153,153,153,33,64,113,61,10,215,163,112,205,63]}"
```
Error:
```
error: Found argument '{"root":"0xd889ae8b2b8807ac5845cee7bb8db1996e01554f88df2040353ea0a582df392f","proof":["0x70997831bb3b0a5eb10dd2c55972f35bcbe781d014ad53811d31743450587292","0x91cfc94f5515165bb59ff651be8bbf3058865f393d78bb102ff686b1782ce360","0xccfd66ee8c896087af65b4ae3f577f86ef82092168e61cbf7f08d3951ef1852a","0xff00158da139d439fade205829bfd1153da3b32b45789f567c0898d563d2abce"],"number_of_leaves":10,"leaf_index":1,"leaf":[1,0,0,0,0,0,0,0,1,100,50,48,50,50,45,48,51,45,48,52,84,48,53,58,48,54,58,48,55,43,48,48,58,48,48,28,97,99,116,111,114,95,49,1,0,0,0,0,154,153,153,153,153,153,33,64,113,61,10,215,163,112,205,63]}' which wasn't expected, or isn't valid in this context
```

Reply:
yes because you have two arguments: Alice and the Proof, but you only need to pass one.

Reply:
`true`.
Sorry, my bad.

Reply:
`true`.
Sorry, my bad.